docs/reference/settings/security-settings.asciidoc

```diff
@@ -239,6 +239,12 @@ This setting is multivalued; you can specify multiple user contexts.
 Required to operate in user template mode. If `user_search.base_dn` is specified, 
 this setting is not valid. For more information on
 the different modes, see {xpack-ref}/ldap-realm.html[LDAP realms].
+
+`authorization_realms`::
+The names of the realms that should be consulted for delegate authorization.
+If this setting is used, then the LDAP realm does not perform role mapping and
+instead loads the user from the listed realms.
```

Comment:
Should we also mention that the order of the items in this value is important? i.e. the realms are tried in the order specified here?